### PR TITLE
Allow early version of X7 firmware Eeprom to migrate to new variant

### PR DESCRIPTION
**Pull request by 3djc**

### CODE REVIEW

File: radio/src/storage/eeprom_rlc.cpp [hunk 1]
```diff
@@ -826,6 +826,14 @@ bool eeLoadGeneral()
     }
   }
 
+  #if defined(PCBX7)
+  if (g_eeGeneral.variant == 0) {
+    TRACE("Pre release EEPROM detected, variant %d instead of %d for X7 radio. Loading anyway", g_eeGeneral.variant, EEPROM_VARIANT);
+    g_eeGeneral.variant = EEPROM_VARIANT;
```
Comment (kilrah) [inline radio/src/storage/eeprom_rlc.cpp]: `storageDirty(EE_GENERAL);`

File: radio/src/storage/eeprom_rlc.cpp [hunk 2]
```diff
@@ -826,6 +826,14 @@ bool eeLoadGeneral()
     }
   }
 
+  #if defined(PCBX7)
```
Comment (bsongis) [inline radio/src/storage/eeprom_rlc.cpp]: indentation problem here

**Comment by bsongis** [comment]
yes you are right, it's better with your change

File: radio/src/storage/eeprom_rlc.cpp (hunk 1 above)
Comment (bsongis) [inline radio/src/storage/eeprom_rlc.cpp]: perhaps not needed as we write the unexpected shutdown bit just after. perhaps I am wrong though, an eeDirty wouldn't hurt

File: radio/src/storage/eeprom_rlc.cpp [hunk 3]
```diff
@@ -827,11 +827,12 @@ bool eeLoadGeneral()
   }
 
   #if defined(PCBX7)
```
Comment (bsongis) [inline radio/src/storage/eeprom_rlc.cpp]: it's worse, everything 2 spaces

**Comment by bsongis** [comment]
👍

**Pull request merged**